content/well-architected-framework/docs/docs/optimize-systems/lifecycle-management/tag-cloud-resources.mdx

```diff
@@ -0,0 +1,159 @@
+---
+page_title: Tag cloud resources
+description: Implement cloud resource tagging best practices with Terraform for AWS, Azure, and GCP. Learn to automate tags, enforce policies, and optimize cost allocation using infrastructure as code.
+---
+
+# Tag cloud resources
+
+Managing thousands of cloud resources across regions, environments, and teams is complex without a systematic approach. Tags are key-value pairs that help you manage, identify, organize, locate, and filter resources. It is important to have a clear, well-defined cloud resource tagging strategy. You can also use tags to track cost allocation and usage, and automate resource management tasks.
```

Comment:
```suggestion
Managing thousands of cloud resources across regions, environments, and teams is complex. Tags are key-value pairs that help you manage, identify, organize, locate, and filter resources. It is important to have a clear, well-defined cloud resource tagging strategy. You can also use tags to track cost allocation and usage, and automate resource management tasks.
```
Having it at the end sounds awkward. Could also re-order a bit but I think it stands well on its own:
`Managing thousands of cloud resources across regions, environments, and teams without a systematic approach, is complex.`